os/arch/arm/src/common/up_cpuidle.c

```diff
@@ -76,32 +76,21 @@
  ****************************************************************************/
 
 /****************************************************************************
- * Name: up_idle
+ * Name: up_cpuidle
```

Comment:
Why do you change API name? Will you use ```up_idle``` with different operation?

Reply:
Have removed up_idle as it was not getting used.
Instead when the idle task is running & no PM framework is present, up_cpuidle() puts system in WFI mode